Client/Frontend/Browser/FaviconManager.swift

```diff
@@ -17,7 +17,7 @@ class FaviconManager : BrowserHelper {
         if let path = NSBundle.mainBundle().pathForResource("Favicons", ofType: "js") {
             if let source = NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding, error: nil) as? String {
                 var userScript = WKUserScript(source: source, injectionTime: WKUserScriptInjectionTime.AtDocumentEnd, forMainFrameOnly: true)
-                browser.webView.configuration.userContentController.addUserScript(userScript)
+                browser.webView?.configuration.userContentController.addUserScript(userScript)
```

Comment:
Maybe worth using `!` instead of `?` in the helpers? Having an optional `webView` opens up a new set of bugs where things might be "skipped" if the `webView` isn't created yet. We should probably strongly enforce that `Browser`s have a valid `webView` for one-off events like this (when helpers are created) -- otherwise, this init does nothing, and the user scripts never get set on the `webView`, and the browser is gimped.